Created Travis YAML with conditional scripting based on branch name

W3stside opened the pull request:
1. /contract(s)/**/** = Contract testing
2. /anythingElse/ = npm run build:prod or dev based on branch

W3stside committed:
Created Travis YAML with conditional scripting based on branch name
1. /contract(s)/**/** = Contract testing
2. /anythingElse/ = npm run build:prod or dev based on branch

Velenir commented on .travis.yml:
```diff
@@ -0,0 +1,21 @@
+#IF on contract branch - start testRPC || ELSE - move on  
+before_script: 
+  - cd $TRAVIS_BUILD_DIR
+  - "chmod +x ./BranchPreScriptCheck.sh && ./BranchPreScriptCheck.sh"
```
I wonder if scripts can be run without `chmod`, just `"sudo ./BranchPreScriptCheck.sh"` or `"sudo $(which bash) ./BranchPreScriptCheck.sh"`

Velenir commented on .travis.yml:
```diff
@@ -0,0 +1,21 @@
+dist: trusty
+language: node_js
+node_js:
+  - '8'
```
Will it work with node `9`?

W3stside committed:
adding Node9 compatibility

W3stside committed:
updated .gitattributes to leave Brittany alone

And by Brittany I mean all image formats due to: [this](https://github.com/angular-fullstack/generator-angular-fullstack/issues/1382)
